### DISCUSSION TITLE
Invalid field directory

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#14931] [Q&A] [answered]
I have tried this cd storage/shared/Android/data/com.mojang.minecraftpe/files/games/com.mojang/development_behavior_packs
And it says this directory is not found on android, I have give app permission to do this how can I do this?

**Comment by agnostic-apollo** [comment, marked as answer]
Firstly, you should be using `~/storage/...`.

Secondly, if using Android `>= 11` , then you can't access `Android/data` directories of other apps as per AOSP. You can try `termux-api` `saf` apis.

https://github.com/termux/termux-app/issues/1952

https://github.com/termux/termux-app/issues/3117

https://developer.android.com/training/data-storage/manage-all-files
